README.md

```diff
@@ -1,24 +1,14 @@
-# README
+# Memebers Only Application
```

Comment:
# Members Only (typo in the word Memebers)

Reply:
Oh thank you Ryan. I'm going to fix it right away.